src/util/mod.rs

```diff
@@ -1,13 +1,18 @@
-use std::{env, sync::Once};
+use std::{env, str::FromStr, sync::Once};
 
 use electrum_client::ElectrumApi;
-use elements::encode::Decodable;
+use elements::{encode::Decodable, hex::ToHex};
+use lightning_invoice::{Bolt11Invoice, RouteHintHop};
 
 use crate::{error::Error, network::electrum::ElectrumConfig};
 
 pub mod ec;
 pub mod secrets;
 
+const ENDPOINT: &str = "https://api.testnet.boltz.exchange";
+const MAGIC_ROUTING_HINT_CONSTANT: &str = "0846c900051c0000";
+const LBTC_ASSET_HASH: &str = "144c654344aa716d6f3abcc1ca90e5641e4e2a7f633bc09fe3baf64585819a49";
```

Comment:
The L-BTC asset hash depends on the network

Reply:
I only found one asset hash from your doc. Added a placeholder for mainnet asset hash. Can be easily updated when I have it.